nixos/modules/services/misc/etebase-server.nix

```
@@ -166,7 +166,7 @@ in
       } ''
         makeWrapper ${pythonEnv}/bin/etebase-server \
           $out/bin/etebase-server \
-          --run "cd ${cfg.dataDir}" \
+          --chdir ${cfg.dataDir} \
```

haven't tested the changes, but they look correct at first glance. this one in particular could benefit from escaping though:
```suggestion
          --chdir ${escapeShellArg cfg.dataDir} \
```

---

Fixed, thanks. That seems to be the only one in `nixos` that comes from arbitrary user input.